src/bot_commands.c

```diff
@@ -2108,6 +2133,59 @@ static void FrogbotsDisable(void)
 	}
 }
 
+static void FrogbotsGoMarker(void)
```

Comment:
If you wish to break out changes from the last commit you can do something like this:
```
# y to extract hunk, s to split it smaller, n to keep it, once you've found hunk to extract, q to skip the rest
git reset -p HEAD^ -- src/bot_commands.c 
git commit --amend # updates the commit without the extracted hunk, not(!) with -a
# now your workspace should have the extracted change so just commit that
git commit -a -m "mah change"
# as you may have more changes to extract, it's easiest to just move the big commit to top of branch
git rebase -i origin/master # shuffle around the lines to change commit order
```

Reply:
Also why does this even exist? Couldn't you just modify `/botcmd goto <marker#>` (`FrogbotGoto()`) to default to finding nearest marker instead of erroring if marker not passed?